docs/source/deploy_groups.rst

```diff
@@ -87,6 +87,11 @@ Existing service
 
 Alternatively to a jenkins pipeline, you can use ``paasta mark-for-deployment`` with the ``--deploy-group`` flag to manually mark each of your deploy groups.
 
+String interpolation
+--------------------
+
+Deploy groups support string interpolation for the following facts: ``cluster``, ``instance`` and ``service``. String interpolation works by surrounding the fact's name with braces (``{}``) in the ``deploy_group`` field -- this is python's ``string.format`` syntax. You must still specify explicit deploy groups in your ``deploy.yaml`` however.
```

Comment:
Just call them variables. "facts" is puppety.

I though it was %() ? Can you link to the actual python docs that describe this format? (as apparently I don't fully understand how to use it)